zerver/models.py

```diff
@@ -671,6 +671,9 @@ class Realm(models.Model):  # type: ignore[django-manager-missing] # django-stub
         default=VIDEO_CHAT_PROVIDERS["jitsi_meet"]["id"]
     )
 
+    JITSI_SERVER_SPECIAL_VALUES_MAP = {"default": None}
+    jitsi_server_url = models.TextField(null=True, default=None)
```

Comment:
I think this should be using https://docs.djangoproject.com/en/4.2/ref/models/fields/#urlfield?